guides/WRITING-DOCUMENTATION.md

```diff
@@ -15,176 +15,147 @@ For a component named "Foo", you would have the following:
         ├── Foo.jsx             React component
         ├── Foo.test.jsx        React component tests
         └── Foo.example.jsx     React component used to render an example
+        └── Foo.example.html    HTML example
 ```
 
-# CSS components
+## CSS components
 
 [KSS](https://github.com/kss-node/kss-node) is the primary way documentation is written. They establish what pages get created and the content on those pages. We use KSS to parse CSS comment blocks to extract documentation, and it expects your comment block to follow this particular format:
 
-```css
+```scss
```

Comment:
why change this to scss?

Reply:
I chose SCSS as the files we use in the design system are `.SCSS`